app/controllers/phases_controller.rb

```diff
@@ -162,6 +162,10 @@ def admin_create
     @phase.description = params["phase-desc"]
     @phase.modifiable = true
     if @phase.save
```

Comment:
I would use Active Record Callbacks (e.g. after_save into phase, section and question) to trigger the update into its template! It would abstract that logic from the controller.